src/scribe_data/cli/cli_utils.py

```diff
@@ -36,11 +38,19 @@
 )
 DATA_DIR = Path(DEFAULT_JSON_EXPORT_DIR)
 
-with LANGUAGE_METADATA_FILE.open("r", encoding="utf-8") as file:
-    language_metadata = json.load(file)
+try:
```

Comment:
These try-excepts are fine, but can we not set `language_metadata = {"languages": []}` at the end of them (and the same for data types). If these files are not being loaded in then we have serious problems. If there's not a test for it already - that both of them are read and are in a specific location - then we should write one!

CC @DeleMike and @catreedle for checking for this test :)

Reply:
Given the reaction here, @catreedle, do you want to check the tests to see if we have some for those two files being accessible, and if not you can open a PR with tests for it? :)

Reply:
Sure, allow me some time as I’m still getting familiar with testing. I'll check and get back to you later. :)

Reply:
Sounds good!

Reply:
From what I see, there aren’t any specific tests to verify whether both language_metadata.json and data_type_metadata.json are being read, but the tests depend on these files. Deleting one causes an error during test runs. I've written tests to check the accessibility of these files, but they won’t run if the files are missing. I’m unsure about adding these tests and could use some help with this. :)

Reply:
I guess I could put it in a new test file for this if it's still needed?

Reply:
Yes, I guess you could but wait for a final go-ahead :)

Reply:
We can have two minor tests just like you described, @catreedle :) Feel free to put them in an appropriate directory/file that's for the whole application. I'll move it if needed 😊

Reply:
added a PR [here](https://github.com/scribe-org/Scribe-Data/pull/343).  Looking forward to the review. :)